### ISSUE TITLE
an onboarding tour to ecli-api containers

### ISSUE THREAD

**Issue by olebrussels** [openjusticebe/ecli-api#11]
hey @tintamarre i cloned the ecli-api repo locally,   run the docker compose up. i have 4  or 5 docker containers locally in my computer, but i miss implementing this last two commands:

- docker-compose exec php php artisan migrate

- Import db from a dump

before i do that , would it be possible to get a _guided tour_ of the whole system and see what can i do  with it. this is totally unexplored grounds for me and i am a bit lost. 

Thanks!

**Comment by tintamarre** [comment]
Yes, of course. I did a guided tour for **Omdat** and plan to do one for this repo. I will try to do it soon.

I keep you posted 📫

**Comment by olebrussels** [comment]
thanks @tintamarre ! i would like to see the guide tour for omdat as well where is it?  i dont see it in the omdat repo

**Comment by tintamarre** [comment]
You can find the link on slack #openjustice-application
